python/pfs/drp/stella/calibrateWavelengthsTask.py

```diff
@@ -315,16 +355,24 @@ def fitWavelengthSolution(self, spec, detectorMap, rng=np.random):
             else:
                 dy = -int(-dy)
                 spec.wavelength[dy:] = spec.wavelength[:-dy]
-
+                
+        detectorMap.setWavelength(fiberId, rows, spec.wavelength)
         diff = detectorMap.getWavelength(fiberId) - spec.wavelength
         self.log.info("Fiber %d: wavelength correction %f +/- %f nm" % (fiberId, diff.mean(), diff.std()))
-        detectorMap.setWavelength(fiberId, rows, spec.wavelength)
```

Comment:
Moving this line makes the above log line useless, because now `detectorMap.getWavelength(fiberId)` equals `spec.wavelength`.